dojo/db_migrations/0008_finding_mitigation.py

```diff
@@ -0,0 +1,36 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dojo', '0007_note_additions'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='finding',
+            name='is_Mitigated',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AlterField(
+            model_name='child_rule',
```

Comment:
All of these changes referencing rules were not changed in this PR. These changes have already been accommodated in the 0006 migration file as well. For some reason, they were not applied when that file ran. They cannot be in both places though.